app/src/main/kotlin/com/wire/android/ui/WireActivityState.kt

```diff
@@ -18,10 +18,11 @@
 
 package com.wire.android.ui
 
-sealed class WireActivityState {
+sealed class
+WireActivityState {
```

Comment:
```suggestion
sealed class WireActivityState {
```

Out of the scope of this PR:
If anything were to be changed here, I'd make it an `interface` instead of a `class`, as it doesn't have any fields